src/main/java/com/google/api/codegen/transformer/java/JavaSurfaceNamer.java

```diff
@@ -41,6 +42,15 @@ public JavaSurfaceNamer() {
   }
 
   @Override
+  public boolean shouldImportRequestObjectParamElementType(Field field) {
+    if (Field.IS_MAP.apply(field)) {
```

Comment:
Does this check only apply to Java? If not we can move this up to the SurfaceNamer.